### PR TITLE
dotnet nuget verify doc

### LINKED ISSUE #20701: add dotnet nuget verify command to dotnet docs
We [`(NuGet.Client)`](https://github.com/NuGet/NuGet.Client) team completed development of NuGet side changes for `dotnet nuget verify` command. This issue is created to add docs to `dotnet` repo.

https://github.com/dotnet/sdk/issues/13702

[Edited by gewarren to add doc details for the general section of the docs where this should land.]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6a91b44f-cab9-7657-9dd1-b913a8e021d0
* Version Independent ID: 0a6371bc-9f43-0bf9-7ec7-fc5d6331c2c8
* Content: [dotnet nuget delete command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-delete)
* Content Source: [docs/core/tools/dotnet-nuget-delete.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-nuget-delete.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @karann-msft
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by kartheekp-ms**
## Summary

`dotnet 5.0.100-rc.2.x` SDK has added `dotnet nuget verify` command.

Happy to iterate to make the doc better based on feedback.

Fixes #20701

### CODE REVIEW

File: docs/core/tools/dotnet-nuget-verify.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  Specifies the file path to the package(s) to be verified. Multiple position arguments can be passed in to verify multiple packages.
+
+## Options
+
+- **`--all`**
+
+  Specifies that all verifications possible should be performed on the package(s). By default, only `signatures` are verified.
+
+> [!NOTE]
+> This command currently supports only `signature` verification. Additional quality checks shall be added in the future.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-verify.md:43]: Suggest we delete promise of future functionality -- See Future product plans or promises in https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-channel-guidance?branch=master

```suggestion
> This command currently supports only `signature` verification.
```

File: docs/core/tools/dotnet-nuget-verify.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+  Sets the MSBuild verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`. The default is `normal`.
+
+* **`-h|--help`**
+
+  Prints out a short help for the command.
+
+## Examples
+
+- Verifies *foo.nupkg*:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-verify.md:59]:
```suggestion
- Verify *foo.nupkg*:
```

File: docs/core/tools/dotnet-nuget-verify.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  dotnet nuget verify foo.nupkg
+  ```
+
+- Verifies multiple NuGet packages *foo.nupkg* and *all .nupkg files in the directory specified*:
+
+  ```dotnetcli
+  dotnet nuget verify foo.nupkg c:\mydir\*.nupkg
+  ```
+
+- Verifies *foo.nupkg* signature matches with the specified certificate fingerprint:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-verify.md:71]:
```suggestion
- Verify *foo.nupkg* signature matches with the specified certificate fingerprint:
```

File: docs/core/tools/dotnet-nuget-verify.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+  ```dotnetcli
+  dotnet nuget verify foo.nupkg --certificate-fingerprint CE40881FF5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E039
+  ```
+
+- Verifies *foo.nupkg* signature matches with one of the specified certificate fingerprints:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-verify.md:77]:
```suggestion
- Verify *foo.nupkg* signature matches with one of the specified certificate fingerprints:
```

**Review by tdykstra — APPROVED** [review]
This looks great, I have just a few suggestions.

**Comment by kartheekp-ms** [comment]
> This looks great, I have just a few suggestions

Thank you @tdykstra for the feedback. I have committed all suggestions. Can we merge this PR now or wait for additional approvals?

**Pull request merged**